selenium/che-selenium-core/src/main/java/org/eclipse/che/selenium/core/client/TestGitHubRepository.java

```diff
@@ -67,16 +68,37 @@ public String getName() {
     return repoName;
   }
 
+  public String getSha1(String branchName) throws IOException {
+    return ghRepo.getBranch(branchName).getSHA1();
+  }
+
   /**
-   * Creates reference to branch, tag, ... from master branch.
+   * Creates reference to the new branch with {@code branchName} from default branch.
    *
-   * @param refName is a name of branch, tag, etc
+   * @param branchName is a name of new branch
    * @return reference to the new branch
    * @throws IOException
    */
-  public GHRef createBranchFromMaster(String refName) throws IOException {
-    GHRef master = ghRepo.getRef("heads/master");
-    return ghRepo.createRef("refs/heads/" + refName, master.getObject().getSha());
+  public GHRef createBranch(String branchName) throws IOException {
+    GHRef defaultBranch = ghRepo.getRef("heads/" + ghRepo.getDefaultBranch());
```

Comment:
Please create public method which will be return the default branch
```
GHRef public getDefaultBranch(){
return ghRepo.getRef("heads/" + ghRepo.getDefaultBranch());
}
```

Reply:
Which test needs it?

Reply:
He can create private method. Multiple using the same row
```
 GHRef defaultBranch = ghRepo.getRef("heads/" + ghRepo.getDefaultBranch());
```
looks not very good

Reply:
Method name `getReferenceToDefaultBranch()` is more relevant to the returning value, IMHO

Reply:
I agree